Journaley/Utilities/PostMarkdownParser.cs

```diff
@@ -0,0 +1,114 @@
+﻿namespace Journaley.Utilities
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Text.RegularExpressions;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Helper class for fixing MarkdownDeep parsed HTML.
+    /// </summary>
+    public class PostMarkdownParser
+    {
+        /// <summary>
+        /// Perform fixes after MarkdownDeep parsing for publishing
+        /// to Journaley.
+        /// </summary>
+        /// <param name="formattedString">Formatted HTML string</param>
+        /// <returns>Properly formatted HTML string for publishing.</returns>
+        public static string PostMarkdown(string formattedString)
+        {
+            // First convert all code blocks using <p> into <pre>
+            // which is used in almost all implementations of
+            // Markdown, even in the original Markdown specs.
+            formattedString = Regex.Replace(formattedString, @"<p><code>", "<pre><code>", RegexOptions.Multiline);
+            formattedString = Regex.Replace(formattedString, @"</code></p>", "</code></pre>", RegexOptions.Multiline);
+
+            System.Text.StringBuilder builder = new System.Text.StringBuilder();
```

Comment:
Don't need the `System.Text.` part. You can just say `StringBuilder` because you already added `using System.Text` above.